src/login/message_server.cpp

```diff
@@ -251,7 +251,7 @@ void message_server_init()
 
     auto server = fmt::format("tcp://{}:{}",
         settings::get<std::string>("network.ZMQ_IP"),
-        settings::get<std::string>("network.ZMQ_PORT"));
+        settings::get<uint16>("network.ZMQ_PORT"));
```

Comment:
This part shouldn't be needed, the get function should do a nice conversion to string if asked

Reply:
amended